src/chart/Bubble.js

```diff
@@ -35,6 +36,23 @@
         return retVal;
     };
 
+    Bubble.prototype.enter = function () {
+        if (this.useTooltip()) {
+            if (!this._parentOverlay && this._parentWidget._parentOverlay) {
+                this._parentOverlay = this._parentWidget._parentOverlay;  //This is because charts do not get an overlay div when they are inside the MultiChart
+            }
```

Comment:
This will fail if the multichart is inside a SVG widget (like Surface) - the "correct" way to get the overlayNode is:
```
var overlayNode = this.locateOverlayNode();
```